src/server/node.rs

```diff
@@ -167,7 +167,11 @@ where
         let mut store_id = self.check_store(&engines)?;
         if store_id == INVALID_ID {
             store_id = self.alloc_id()?;
-            debug!("alloc store id"; "store_id" => store_id);
+            if store_id == 0 {
```

Comment:
It should return error instead. `store_id` should not be 0.

Reply:
addressed